Cargo.toml

```diff
@@ -30,6 +30,7 @@ parquet = { version = ">=40, <43", features = ["async"], optional = true }
 rust_decimal = "1.30"
 chrono = "0.4"
 faster-hex = "0.8.0"
+int-enum = "0.5"
```

Comment:
I don't like the idea to introduce a new dep for this job. How about using from/into by hand？ I believe the maintenance burden is not heavy .